.ci/script/docker/package.sh

```diff
@@ -13,8 +13,7 @@ elif [ $# -eq 2 ]; then
     VERSION=$1
     KIBANA_VERSION=$2
     DESTINATION=release/
-    CMD="./mvnw clean verify -B -e && \
```

Comment:
why delete clean here?

Reply:
Good catch

Reply:
`maven clean verify `is used to test, but because I delete the `-DskipTests=true ` in the deploy phase(the test will run there), so the code here is not necessary any more.